pymongo/errors.py

```diff
@@ -171,11 +182,6 @@ def details(self):
         """
         return self.__details
 
-    def __str__(self):
-        output_str = "%s, full error: %s" % (self._message, self.__details)
-        if sys.version_info[0] == 2 and isinstance(output_str, unicode):
-            return output_str.encode('utf-8', errors='replace')
-        return output_str
```

Comment:
Can you move `__str__` back to where it was so that it's not changed in this PR?